features/step_definitions/assertion_steps.rb

```diff
@@ -0,0 +1,11 @@
+Then("I should be on the landing page") do
+  visit root_path
+end
+
+When("I click {string} link") do |link|
+  click_link link
+end
+
+Then("I should see {string}") do |content|
+  expect(page).to have_content content 
+end
```

Comment:
Assertions should be placed in a separate file

Reply:
You are in the assertion_steps.rb